### PR TITLE
Spellchecking construction progress

### PR DESCRIPTION
**Pull request by hex37**
## About The Pull Request
<details>
<summary>
	Exosuit Description improvements.
</summary>
<hr>
Added better exosuit part descriptions and construction instructions for the Durand, Ripley, Firefighter and Gygax.
Updated the player's construction actions for the Durand, Ripley, Firefighter and Gygax to make more grammatical sense.
Updated the text for the commented out deconstruction process for the Durand, Ripley, Firefighter and Gygax, in case we ever want to readd the feature.
<hr>
</details>

## Changelog
:cl:
Added better exosuit part descriptions and construction instructions for the Durand, Ripley, Firefighter and Gygax. Now lists the actual names of the parts being installed.
Updated the player's construction actions for the Durand, Ripley, Firefighter and Gygax to make more grammatical sense.
Updated the text for the commented out deconstruction process for the Durand, Ripley, Firefighter and Gygax, in case we ever want to readd the feature.
/:cl:

### CODE REVIEW

**Comment by hex37** [comment]
Next will be the Odysseus and Phazon.

File: code/game/mecha/mecha_construction/firefighter.dm [hunk 1]
```diff
@@ -31,422 +31,425 @@
...
 		//7
 		list("key"=QUALITY_WELDING,
 			//"backkey"=QUALITY_BOLT_TURNING,
-			"desc"="Internal armor is wrenched but not welded down"),
+			"desc"="The insulation is installed, but segmented. It must be welded together"),
 		//8
 		list("key"=/obj/item/stack/material/plastic,
 			//"backkey"=QUALITY_PRYING,
-			"desc"="Internal armor is secured but needs plastic fitting."),
+			"desc"="The The internal armor is finished, but needs 5 plastic sheets for insulation."),
```
Comment (r4d6) [inline code/game/mecha/mecha_construction/firefighter.dm:62]: You put two `the`

File: code/game/mecha/mecha_construction/firefighter.dm [hunk 2]
```diff
@@ -31,422 +31,425 @@
...
+			"desc"="With the heavy cell mount added, it can now have hydraulic circuits inserted."),
 		//23
 		list("key"=/obj/item/tool_upgrade/augment/cell_mount,
 			//"backkey"=QUALITY_PRYING,
-			"desc"="Mech case is made and needs to start with a cell mount."),
+			"desc"="The basic chassis is assembled. A heavy cell mount is needed to power it."),
 		//24
 		list("key"=QUALITY_SCREW_DRIVING,
-			"desc"="Mech case cover is screwed close.") //So we can successfully keyback to this for getting back tool mods
+			"desc"="The case of the chassis screwed shut. It must be unscrewed to continue.") //So we can successfully keyback to this for getting back tool mods
```
Comment (r4d6) [inline code/game/mecha/mecha_construction/firefighter.dm:125]:
```suggestion
			"desc"="The case of the chassis is screwed shut. It must be unscrewed to continue.") //So we can successfully keyback to this for getting back tool mods
```

**Review by r4d6 — CHANGES_REQUESTED** [review]
I just found two typos in the whole PR. Good job!


... do you think you can also do the handmade bots please?...

File: code/game/mecha/mecha_construction/firefighter.dm (hunk 1 above)
Comment (hex37) [reply code/game/mecha/mecha_construction/firefighter.dm]: Fixed!

File: code/game/mecha/mecha_construction/firefighter.dm (hunk 2 above)
Comment (hex37) [reply code/game/mecha/mecha_construction/firefighter.dm]: Fixed!

**Pull request merged**